setup.cfg

```diff
@@ -100,6 +100,9 @@ plots =
     matplotlib~=3.6.0
     seaborn~=0.11.0
 
+decodingtrust =
```

Comment:
Also add `crfm-helm[decodingtrust]` to `metrics` above:

```
metrics =
    crfm-helm[decodingtrust]
```

Reply:
Thanks for the comment. This has been fixed in 3fc7f79c51bca48bb04cbab2f8bfe3cbdc29cefe.

Reply:
Actually, could you add `crfm-helm[decodingtrust]` after `crfm-helm[plots]` under `all =` below?